terms/immunoAssays/antibodyDilution.json

```diff
@@ -0,0 +1,7 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "sage.annotations-immunoAssays.antibodyDilution-0.0.1",
+    "description": "Ratio of antibody used in an assay",
+    "type": "string"
```

Comment:
Will people be providing this as a fraction, not a decimal?

Reply:
@Aryllen would you comment on this?

Reply:
That's a good question, @karawoo. This is expected to be a ratio like 10:100. If I remember correctly from the conversation when this was being discussed, the researchers tended to use the ratio. Should this be more clear?

Reply:
I think it's fine if it will be familiar to the researchers

Reply:
Cool! Thank you for questioning!